src/main/java/com/Futurecollars/Lekcja10/Zadanie1/PhoneBookExample.java

```diff
@@ -0,0 +1,36 @@
+package com.Futurecollars.Lekcja10.Zadanie1;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class PhoneBookExample {
+    public static void main(String[] args) {
+        PhoneBook bookEntry = new PhoneBook();
+        try (BufferedReader br = new BufferedReader(new FileReader("src/main/java/com/Futurecollars/Lekcja10/Zadanie1/data.txt"))) {
```

Comment:
powinieneś to wydzielić do jakiejś metody w stylu, load phoneBook(String filePath)